### PR TITLE
correcting the docs

### PR DESCRIPTION
**Pull request by soma2000-lang**

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -5,13 +5,11 @@
 # dir.
 #
 # The contents of this file are pickled, so don't put values in the namespace
-# that aren't picklable (module imports are okay, they're removed
+# that isn't pickable (module imports are okay, they're removed
```
Comment (rcomer) [inline doc/conf.py:8]: This change is incorrect. “values” are plural.

File: doc/conf.py [hunk 2]
```diff
@@ -5,13 +5,11 @@
...
 # automatically).
 #
 # All configuration values have a default value; values that are commented out
 # serve to show the default value.
-
-import logging
```
Comment (rcomer) [inline doc/conf.py:14]: The `logging` module is used below (see flake8) comment.  So we can not remove it.

**Review by rcomer — CHANGES_REQUESTED** [review]
Thank you for your contribution @soma2000-lang, but I do not think we can use either of your suggestions.

File: doc/conf.py (hunk 1 above)
Comment (ksunden) [reply doc/conf.py:8]: Additionally "picklable" is what we mean, as in "able to use the `pickle` module to serialize"

It is a bit of a jargon term, but is used in this context often, including in the pickle module docs:

https://docs.python.org/3/library/pickle.html

**Comment by jklymak** [comment]
Thanks @soma2000-lang - but these changes aren't correct, so no point in keeping this open. We hope to see you again!

**Pull request closed without merging**